webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -12695,14 +12694,20 @@ var userOverride = {
                 return conversationDetail;
             };
             _this.checkBotDetail = function (userId) {
-                window.Applozic.ALApiService.ajax({
+                var authKey = JSON.parse(
```

Comment:
Is this the same which is used in any other api?

Reply:
yes.

Reply:
can you share any code reference for that

Reply:
how the headers are being set?

Reply:
I mean any other place from where auth header is fetched from storage

Reply:
Ratified